Kickstarter-iOS/Views/Cells/ProjectPamphletMainCell.swift

```diff
@@ -231,16 +229,6 @@ internal final class ProjectPamphletMainCell: UITableViewCell, ValueCell {
     _ = self.statsStackView
       |> UIStackView.lens.isAccessibilityElement .~ true
       |> UIStackView.lens.backgroundColor .~ .white
-
-    _ = self.youreABackerContainerView
```

Comment:
The reason I'm asking is because I've noticed @ifbarrera 's earlier comment and it would all be cool in case we're using the `CTA` button but since that is feature flagged it is potentially possible we will still want to show this screen with the old vertical rewards on it (maybe as an A/B test).

So in case of CTA removing this button makes sense because the CTA button itself is very descriptive and replaces this functionality, but

in case of the old design, users won't have any way of telling the state from the UI

Just wanted to double check

Reply:
Removing the `You're a backer` from the screen was not intentional. It will be removed from the once we redesign the rest of the Project screen. The confusion is my fault; the mockups were just using a repeated project page to give context. They did not represent state but must have been bundled in when engineering tasks were made.

I was fine with letting it drop-off but you raise a good point RE still keeping vertical rewards around in some circumstances. Let's re-add the `You're a backer` for now then, please @cdolm92.

Reply:
cool 👍🏾

Reply:
@dannyalright @cdolm92 I think we could actually add this back in a way that would allow us to conditionally show/hide it depending on whether the `featureNativeCheckout` feature flag is enabled 🤔 maybe that's the happy medium?

Reply:
fine with me! Good idea